packages/graphql/tests/schema/directives/relationship-aggregate.test.ts

```diff
@@ -0,0 +1,650 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { gql } from "graphql-tag";
+import { Neo4jGraphQL } from "../../../src";
+import type { GraphQLFieldMap, GraphQLObjectType } from "graphql";
+import { lexicographicSortSchema } from "graphql";
+import { printSchemaWithDirectives } from "@graphql-tools/utils";
+
+describe("@relationship directive, aggregate argument", () => {
```

Comment:
I know this is a no-op with interfaces/unions but is it worth logging a warning if someone tries to use this with them and having a test case for that? We don't want our users expecting aggregations and not getting them!